Header header

float64 latitude
float64 longitude
float64 absolute_altitude
float64 relative_altitude

geometry_msgs/Vector3 velocity_linear

float64 drone_roll
float64 drone_pitch
float64 drone_yaw

float64 gimbal_roll
float64 gimbal_pitch
float64 gimbal_yaw